dipy/tracking/tests/test_utils.py

```diff
@@ -258,46 +207,46 @@ def _target(target_f, streamlines, voxel_both_true, voxel_one_true,
 
     # Both pass though
     mask[voxel_both_true] = True
-    new = list(target_f(streamlines, mask, affine=affine))
+    new = list(target_f(streamlines, affine, mask))
     npt.assert_equal(len(new), 2)
-    new = list(target_f(streamlines, mask, affine=affine, include=False))
+    new = list(target_f(streamlines, affine, mask, include=False))
     npt.assert_equal(len(new), 0)
 
     # only first
     mask[:] = False
     mask[voxel_one_true] = True
-    new = list(target_f(streamlines, mask, affine=affine))
+    new = list(target_f(streamlines, affine, mask))
     npt.assert_equal(len(new), 1)
     assert_true(new[0] is streamlines[0])
-    new = list(target_f(streamlines, mask, affine=affine, include=False))
+    new = list(target_f(streamlines, affine, mask, include=False))
     npt.assert_equal(len(new), 1)
     assert_true(new[0] is streamlines[1])
 
     # Test that bad points raise a value error
     if test_bad_points:
         bad_sl = streamlines + [np.array([[10.0, 10.0, 10.0]])]
-        new = target_f(bad_sl, mask, affine=affine)
+        new = target_f(bad_sl, affine, mask)
         npt.assert_raises(ValueError, list, new)
         bad_sl = streamlines + [-np.array([[10.0, 10.0, 10.0]])]
-        new = target_f(bad_sl, mask, affine=affine)
+        new = target_f(bad_sl, affine, mask)
         npt.assert_raises(ValueError, list, new)
 
     # Test smaller voxels
     affine = np.array([[.3, 0, 0, 0],
                        [0, .2, 0, 0],
                        [0, 0, .4, 0],
                        [0, 0, 0, 1]])
-    streamlines = list(move_streamlines(streamlines, affine))
-    new = list(target_f(streamlines, mask, affine=affine))
+    streamlines = list(transform_streamlines(streamlines, affine))
```

Comment:
you should remove `list` since `transform_streamlines` return a list and not an iterator like `move_streamlines`